src/test/java/no/unit/nva/cristin/projects/FetchCristinProjectsTest.java

```diff
@@ -231,7 +229,13 @@ void readerThrowsIoExceptionWhenReadingInvalidJson() {
     void handlerReturnsProjectsWrapperWithAllMetadataButEmptyHitsArrayWhenNoMatchesAreFoundInCristin()
         throws Exception {
 
-        modifyHttpResponseToClient(EMPTY_LIST_STRING, generateHeaders(ZERO_VALUE, LINK_EXAMPLE_VALUE));
+        cristinApiClientStub = spy(cristinApiClientStub);
+        doReturn(new HttpResponseStub(EMPTY_LIST_STRING, HttpURLConnection.HTTP_OK,
+            generateHeaders(ZERO_VALUE, LINK_EXAMPLE_VALUE)))
+            .when(cristinApiClientStub).queryProjects(any(), any());
+        doReturn(Collections.emptyList()).when(cristinApiClientStub).fetchQueryResultsOneByOne(any());
```

Comment:
Since `fetchQueryResultsOneByOne` is mocked with a default list of projects in `CristinApiClientStub` we have to explicitly return `emptyList()`.